ui/src/reducers/apps.js

```diff
@@ -0,0 +1,6 @@
+export default function apps(state = [], action) {
+  switch (action.type) {
+    default:
```

Comment:
`default` action only action defined for now, as our app currently has no need for a reducer function yet